mycroft/skills/skill_manager.py

```diff
@@ -172,36 +233,36 @@ def _load_on_startup(self):
 
     def _reload_modified_skills(self):
         """Handle reload of recently changed skill(s)"""
-        reload_occured = False
         for skill_dir in self._get_skill_directories():
             try:
                 skill_loader = self.skill_loaders.get(skill_dir)
                 if skill_loader is not None and skill_loader.reload_needed():
                     skill_loader.reload()
-                    reload_occured = True
+                    if skill_loader.instance:
+                        self.upload_queue.put(skill_loader)
             except Exception:
                 LOG.exception('Unhandled exception occured while '
                               'reloading {}'.format(skill_dir))
 
-        if reload_occured:
-            # If a reload occured a skill gid may have changed.
-            self.skill_updater.post_manifest(reload_skills_manifest=True)
-
     def _load_new_skills(self):
         """Handle load of skills installed since startup."""
         for skill_dir in self._get_skill_directories():
             if skill_dir not in self.skill_loaders:
-                self._load_skill(skill_dir)
+                sl = self._load_skill(skill_dir)
```

Comment:
I want to stay away from over-abbreviated variable names like this one.  Could we call this "loader" or "skill_loader"?